common/test/db_fixtures/grades.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "pk": 1,
+        "model": "grades.persistentgradesenabledflag",
+        "fields": {
+            "enabled": true,
+            "enabled_for_all_courses": true,
+            "change_date": "2016-09-01"
+        }
+    }
+]
```

Comment:
I guess we can confirm once we implement the [logging ticket](https://openedx.atlassian.net/browse/TNL-5398).  I updated it to add a task to verify bokchoy once we have logging.